### ISSUE TITLE
Don't use sbtx in the CI

### ISSUE THREAD

**Issue by mdedetrich** [apache/incubator-pekko#59]
In this PR https://github.com/apache/incubator-pekko/pull/53#discussion_r1024985382 to get CI running we ended up having to use sbtx instead of plain sbt due to having to upgrade a github action. It turns out that the change to sbtx was that in the older versions of the github action, it was aliasing `sbt` to `sbtx` (see https://discord.com/channels/632150470000902164/922600050989875282/1042858091387043872). We should investigate into using pure sbt rather than sbtx since sbtx isn't really supported that much anymore

**Comment by pjfanning** [comment]
sbtx is just a script to run sbt commands. It still respects the projects/build.properties and the actual sbt runtime that is needed to run the sbt version are still downloaded as they would be with the standard sbt script.

**Comment by mdedetrich** [comment]
This I understand, the reason behind this issue is that sbtx is an ancient script (originally created by paulp) which was mainly created because in the old days of sbt there was a lot of ceremony in setting up sbt, something that is now largely irrelevant because a lot of that functionality was moved into official sbt. If you read the linked discord ticket you will also gets similar sentiment regarding the usage of sbtx.

Due to this (and hence the main motivator for this ticket), sbtx is not installed anymore on distros/Windows/OSX. Everyone just downloads/installs/sets up pure SBT which comes to the final point which I want to get to, which is simulating the CI on your local machine (a problem that I experienced when I worked on non trivial PR's).

Note that this ticket is not high priority, but its definitely ideal to use `sbt` rather than `sbtx` for these reasons.
